src/Symfony/Component/DependencyInjection/Tests/Dumper/XmlDumperTest.php

```diff
@@ -287,6 +291,34 @@ public function testDumpHandlesEnumeration()
         $this->assertEquals(file_get_contents(self::$fixturesPath.'/xml/services_with_enumeration.xml'), $dumper->dump());
     }
 
+    // https://github.com/symfony/symfony/issues/48178
+    /**
+     * @requires PHP 8.1
+     * @dataProvider provideDefaultClasses
```

Comment:
we don't reference github in the source (and fabbot is right on this one)
```suggestion
    /**
     * @requires PHP 8.1
     *
     * @dataProvider provideDefaultClasses
```

Reply:
Okay. Do we have to remove all GitHub issues references in tests ? ex: https://github.com/symfony/symfony/tree/6.4/src/Symfony/Component/Form/Tests/ChoiceList/Factory/PropertyAccessDecoratorTest.php#L97

Reply:
we could indeed